.github/workflows/tag_deploy_github-only.yml

```diff
@@ -0,0 +1,67 @@
+# Deploy GitHub release when a SemVer tag is pushed
+# ------------------------------------------------------------------------------
+#
+#             NOTICE: **This file is maintained with puppetsync**
+#
+# This file is updated automatically as part of a standardized asset baseline.
+#
+# The next baseline sync will overwrite any local changes to this file!
+#
+# ------------------------------------------------------------------------------
+#
+# NOTES:
+#
+# * The CHANGLOG text is altered to remove RPM-style date headers, which don't
+#   render well as markdown on the GitHub release pages
+#
+---
+name: 'Tag: Release to GitHub'
+
+on:
+  push:
+    tags:
+      - '[0-9]+\.[0-9]+\.[0-9]+'
+
+jobs:
+  create-github-release:
+    name: Deploy GitHub Release
+    if: github.repository_owner == 'simp'
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
+          clean: true
+          fetch-depth: 0
+      - name: Get tag & annotation info (${{github.ref}})
```

Comment:
Is there a reason we are not using pkg:create_tag_changelog for the tag annotation?

Reply:
@lnemsick-simp Presumably, `rake pkg:create_tag_changelog` would have been used to create the tag annotation that is triggering the release.